server/http_handler.go

```diff
@@ -303,6 +303,25 @@ func (t *tikvHandlerTool) getAllHistoryDDL() ([]*model.Job, error) {
 	return jobs, nil
 }
 
+func (t *tikvHandlerTool) resignDDLOwner() error {
+	s, err := session.CreateSession(t.store.(kv.Storage))
+	if err != nil {
+		return errors.Trace(err)
+	}
+
+	if s != nil {
+		defer s.Close()
+	}
+
+	dom := domain.GetDomain(s.(sessionctx.Context))
```

Comment:
We can use `session.GetDomain()`.

Reply:
What is the difference? Use session.GetDomain() will look likes:

```
dom, err := session.GetDomain(s.GetStore())
	if err != nil {
		return errors.Trace(err)
	}
```

Reply:
The difference is that we do not need to create a session.